libr/core/cbin.c

```diff
@@ -1029,7 +1029,7 @@ static int srclineCmp(const void *a, const void *b) {
 	if (fa && fb) {
 		return !strcmp (fa, fb);
 	}
-	return (fa && fb && !strcmp (fa, fb));
+	return (a && !b)? 1: -1;
```

Comment:
```
$ cat a.c
int main () {
	printf ("%d %d %d\n",
	strcmp ("a", "b"),
	strcmp ("a", "a"),
	strcmp ("b", "a"));
}
$ ./a.out
-1 0 1
$
```

before it was returning 0, which is wrong, because "a" != NULL, and returning 0 means they are equal. now it returns 1 or -1.. and maybe we can say equal if a == NULL && a == b

Reply:
We could just do ‘return a ==b’. It returns true if they are both null, false otherwise.